### PR TITLE
Extra params

### PR DESCRIPTION
**Pull request by rusefillc**
another safe commit

### CODE REVIEW

File: firmware/boards/f1_dave/build_wideband.sh [hunk 1]
```diff
@@ -5,5 +5,7 @@ set -e
 # back out to the root
 cd ../..
 
+export EXTRA_PARAMS="-DECHO_UART=TRUE"
+
 # build app firmware!
-make -j12 BOARD=f1_dave ECHO_UART=TRUE
+make -j12 BOARD=f1_dave
```
Comment (mck1117) [inline firmware/boards/f1_dave/build_wideband.sh:11]: just do `make ... EXTRA_PARAMS=-DECHO_UART=TRUE`

Comment (mck1117) [reply firmware/boards/f1_dave/build_wideband.sh:11]: why export?

**Comment by rusefillc** [comment]
@dron0gus ^^^

File: firmware/boards/f1_dave/build_wideband.sh (hunk 1 above)
Comment (dron0gus) [reply firmware/boards/f1_dave/build_wideband.sh:11]: Just keeping the same style with RusEFI firmware board build scripts.

Comment (mck1117) [reply firmware/boards/f1_dave/build_wideband.sh:11]: See https://github.com/rusefi/rusefi/pull/4081 for why I don't want to use `export`

**Comment by rusefillc** [comment]
@dron0gus do we care or shall we close it?

**Comment by dron0gus** [comment]
@rusefillc close, please.

**Pull request closed without merging**